app/controllers/Api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,52 @@
+class Api::V1::GreetingsController < ApplicationController
+  before_action :set_greeting, only: %i[show update destroy]
+
+  # GET /greetings
+  def index
+    @greetings = Greeting.all
+
+    render json: @greetings
+  end
+
+  # GET /greetings/1
+  def show
+    render json: @greeting
+  end
+
+  # POST /greetings
+  def create
+    @greeting = Greeting.new(greeting_params)
+
+    if @greeting.save
+      render json: @greeting, status: :created, location: @greeting
```

Comment:
Please create only one action which returns a random greeting. You don't need all the actions for CRUD.